mantle/kola/tests/rhcos/disk.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rhcos
+
+import (
+	"encoding/json"
+	"fmt"
+
+	ignconverter "github.com/coreos/ign-converter"
+	ignv3types "github.com/coreos/ignition/v2/config/v3_0/types"
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform"
+	"github.com/coreos/mantle/platform/conf"
+	"github.com/coreos/mantle/platform/machine/unprivqemu"
+	"github.com/coreos/mantle/util"
+)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Run:         varLibContainers,
+		ClusterSize: 0,
+		Name:        `rhcos.disk.varlibcontainers`,
+		Flags:       []register.Flag{},
+		Distros:     []string{"rhcos"},
```

Comment:
Why just RHCOS?  I'm not aware of a reason this wouldn't work with FCOS too.

Reply:
Good point.  I'll have to figure out where to place this test.

Reply:
I moved the the test to the misc directory so it can run on RHEL CoreOS or Fedora CoreOS.  Tested on both and it passed.